es-topic-indexer/example/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  #- ssh://github.com/utilitywarehouse/shared-kustomize-bases//es-topic-indexer/manifests?ref=v0.0.0
+  - ../manifests
+
+
+# Add prefix to name
+namePrefix: sample-event-
+
+# OpsLevel annotations
+commonAnnotations:
+  "app.uw.systems/description": "Sample events indexer"
+  "app.uw.systems/tier": "tier_4"
```

Comment:
Do we need these here? There are already annotations on the deployment in `es-topic-indexer.yaml`, adding them on everything might just clutter things (e.g. do we really care for/want these on the certificate?)